Mathlib/Data/Nat/Defs.lean

```diff
@@ -978,11 +978,15 @@ lemma mod_two_ne_zero : n % 2 ≠ 0 ↔ n % 2 = 1 := mod_two_not_eq_zero
 lemma div_mod_eq_mod_mul_div (a b c : ℕ) : a / b % c = a % (b * c) / b :=
   (mod_mul_right_div_self a b c).symm
 
-protected lemma lt_div_iff_mul_lt (hdn : d ∣ n) (a : ℕ) : a < n / d ↔ d * a < n := by
+/-- Variant of `Nat.lt_div_iff_mul_lt` (added in the next lean version) that assumes `d ∣ n`. -/
```

Comment:
I of course trust that you'll remember to update this docstring after the release 😉 , but something like this might be slightly more clear in the meantime.
```suggestion
/-- Variant of `Nat.lt_div_iff_mul_lt` (added in Lean 4.16) that assumes `d ∣ n`. -/
```

bors d+